src/cookie.js

```diff
@@ -82,7 +82,10 @@ cookie.set = function (key, value, options) {
     var secure = options.secure || this.defaults.secure ? ';secure' : '';
     if (options.secure === false) secure = '';
 
-    document.cookie = utils.encode(key) + '=' + utils.encode(value) + expires + path + domain + secure;
+    var sameSite = options.sameSite || this.defaults.sameSite
+    sameSite = sameSite === 'None' || sameSite === 'Lax' || sameSite === 'Strict' ? ';sameSite=' + sameSite : '';
```

Comment:
I'm not sure fixing these three options (None, Lax, Strict) is a good idea since we'd need to update the code if new options get introduced. How about just doing `sameSite ? ';sameSite=' + sameSite : ''`, like for path and domain?